ms2/webapp/MS2/lorikeet.lib.xml

```diff
@@ -2,8 +2,7 @@
     <library compileInProductionMode="false">
         <script path="MS2/lorikeet_0.3/css/lorikeet.css"/>
 
-        <script path="MS2/lorikeet_0.3/js/jquery-1.4.2.min.js"/>
```

Comment:
Am I missing where these files are being deleted too?

Reply:
Nope... I wasn't comfortable deleting Lorikeet dependencies until everyone bought off on this approach. Feel free to delete them.

Reply:
I like to delete things, so I did.